### DISCUSSION TITLE
Cut option not available in project

### DISCUSSION THREAD

**Discussion by cwlcampbellx** [keiwando/toonsquid#599] [General]
I have a project that i can cut no issues, but i have another that once drawn the area i want to cut i on;ly get copy/paste no cut option. My other project is fine. Noob here. Any idea what i am missing?

**Comment by keiwando** [comment]
Is the selected drawing layer either a [pixel layer](https://toonsquid.com/handbook/layers/layer_types/#drawing-layers) or a [vector layer](https://toonsquid.com/handbook/layers/layer_types/#vector)? Only those two layer types currently support cutting a selected region out of them.
